apps/zero-runtime-vite-app/src/Slider/ZeroSlider.test.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-filename-extension */
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import Slider from './ZeroSlider';
+
+describe('Slider', () => {
```

Comment:
We should add tests about checking the generated styles, this is what we internally would mostly test. In order to test the functionality of what is added in the PR, I believe we should replicate most of the sx prop tests.

Reply:
Added a snapshot test and basic computed style check on prop change. Since this is just a demo app, I've not included elaborate testing here.

Reply:
Cool, I think we can drop the snapshot test, checking some basic computed styled is enough. This will allow us to test for e.g. the sx prop, conditional styles etc.